fs/server.go

```diff
@@ -195,6 +195,13 @@ func (s *server) Walk(gc go9p.Conn, t *proto.TWalk) (proto.FCall, error) {
 		}
 	}
 
+	if t.Nwname > 0 && t.Wname[0] == "." {
+		c.fids.Store(t.Newfid, info.deriveInfo(file))
+		qids := make([]proto.Qid, 1)
+		qids[0] = file.Stat().Qid
+		return &proto.RWalk{proto.Header{proto.Rwalk, t.Tag}, 1, qids}, nil
+	}
+
```

Comment:
Good change.

I'm wondering if there's any point in storing the new fid/qid since they should already be pointed at the current file, though. Maybe we could just return? 

It's fine either way, this is just a curiosity for me.